src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double upperSide;
+    private double lowerSide;
+    private double height;
+    private double diagonal1;
+    private double diagonal2;
```

Comment:
please remove these 2 fields - we don't need them

Reply:
add